### PR TITLE
Use OkHttp for tests like in NewPipe

### PR DESCRIPTION
**Pull request by wb9688**
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [x] I have tested the API against [NewPipe](https://github.com/TeamNewPipe/NewPipe).
- [x] I agree to create a pull request for [NewPipe](https://github.com/TeamNewPipe/NewPipe) as soon as possible to make it compatible with the changed API.

### CODE REVIEW

File: extractor/build.gradle [hunk 1]
```diff
@@ -8,4 +8,5 @@ dependencies {
     implementation 'org.nibor.autolink:autolink:0.10.0'
 
     testImplementation 'junit:junit:4.13'
+    testImplementation "com.squareup.okhttp3:okhttp:3.12.11"
```
Comment (Stypox) [inline extractor/build.gradle:11]: Can't you use the latest version? Since this only needs to be run in tests, it does not need to be API19-compatible

Comment (wb9688) [reply extractor/build.gradle:11]: Yes, however I think it's a good idea to use the same version in NewPipe and NewPipeExtractor's tests. OkHttp 3.12 will also just still be supported until December 31, 2021.

Comment (Stypox) [reply extractor/build.gradle:11]: Ok, I see

**Comment by Stypox** [comment]
@TobiGr I wasn't able to merge it due to the failing Travis CI, but I already added the PR to the release notes. So just merge ;-)

**Pull request merged**